dsalgo/linked_list.py

```diff
@@ -331,51 +331,71 @@ def skip_i_delete_j(self, i, j):
 		return - return the updated head of the linked list
 		"""
 		'''
-		The Idea: 
+		The Idea:
 			Traverse the Linkedist. Make use of two references - `current` and `previous`.
-			- Skip `i-1` nodes. Keep incrementing the `current`. Mark the `i-1`^th node as `previous`. 
+			- Skip `i-1` nodes. Keep incrementing the `current`. Mark the `i-1`^th node as `previous`.
 			- Delete next `j` nodes. Keep incrementing the `current`.
 			- Connect the `previous.next` to the `current`
 		'''
 		# Edge case - Skip 0 nodes (means Delete all nodes)
 		head = self.head
 		if i == 0:
 			return None
-		
+
 		# Edge case - Delete 0 nodes
 		if j == 0:
 			return head
-		
+
 		# Invalid input
 		if head is None or j < 0 or i < 0:
 			return head
 
 		# Helper references
 		current = head
 		previous = None
-		
+
 		# Traverse - Loop untill there are Nodes available in the LinkedList
 		while current:
-			
+
 			'''skip (i - 1) nodes'''
 			for _ in range(i - 1):
 				if current is None:
 					return head
 				current = current.next
 			previous = current
 			current = current.next
-			
+
 			'''delete next j nodes'''
 			for _ in range(j):
 				if current is None:
 					break
 				next_node = current.next
 				current = next_node
-			
-			'''Connect the `previous.next` to the `current`''' 
+
+			'''Connect the `previous.next` to the `current`'''
 			previous.next = current
-		
+
 		# Loop ends
-		
+
 		return head
 
+	def merge_linked_list(self, list2):
+
+		"""
+		:param: head - head of first linked list
+		:param: list2 - head of second linked list
+		return - return a new merged linked list
+		"""
+
+		sorted_array = []
+		array = self.to_list()
+
+		linked_list = LinkedList()
+		for value in array:
```

Comment:
you can also reduce these steps but it's fine.

Reply:
Do I need to use a diff approach for merge() ?